core/base/src/TROOT.cxx

```diff
@@ -521,6 +521,9 @@ namespace Internal {
    /// a hint for ROOT: it will try to satisfy the request if the execution
    /// scenario allows it. For example, if ROOT is configured to use an external
    /// scheduler, setting a value for 'numthreads' might not have any effect.
+   ///
+   /// \note `EnableImplicitMT(1)` creates a thread-pool of size 1; in order to disable multi-threading,
+   ///       use DisableImplicitMT() (some locks will remain in place as described in EnableThreadSafety()).
```

Comment:
As this is meant for users who look for "how do I disable MT" maybe turn it around?

```suggestion
   /// \note Use `DisableImplicitMT()` to disable multi-threading. `EnableImplicitMT(1)` still creates a thread-pool
   /// of size 1, some locks will remain in place as described in `EnableThreadSafety()`.
```